tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -384,14 +384,14 @@ private void processRequest(HttpServletRequest request, HttpServletResponse resp
                 addGeneralStats(r);
                 send(r, out);
             } else if (what.equals(WHAT_FLAGGED)) {
-                JSONWriter r = newJSONStatus(sm);
+                JSONWriter r = newJSONStatus(request, sm);
```

Comment:
I wonder if these `r` instances should be hoisted up, since they are almost always used.  BUt, not really important.

Reply:
I agree it's too much duplication. processRequest is over 700 lines, so nontrivial to refactor. When we do, good idea to consolidate calls to newJSONStatus